build/generate.mk

```diff
@@ -116,7 +116,9 @@ generate-go: generate-mockgen-installed ## Generates go with 'go generate' (Mock
 generate-bicep-types: generate-node-installed ## Generate Bicep extensibility types
 	@echo "$(ARROW) Generating Bicep extensibility types from OpenAPI specs..."
 	@echo "$(ARROW) Build autorest.bicep..."
-	cd hack/bicep-types-radius/src/autorest.bicep; \
+	cd bicep-types/src/bicep-types; \
+	npm ci && npm run build; \
+	cd ../../../hack/bicep-types-radius/src/autorest.bicep; \
```

Comment:
fmi.. more info on reason for this path change?

Reply:
Just some root directory details. Earlier we start with cd-ing into the hack folder as the working directory. But now we need to build the `bicep-types` library first so we're cd-ing into that as the working directory. `cd ../../../hack/bicep-types-radius/src/autorest.bicep;` is how we go from the bicep-types folder to the autorest folder to change the execution directory